R/task-retry.R

```diff
@@ -23,8 +23,16 @@
 ##'
 ##' @export
 ##' @return New identifiers for the retried tasks
-task_retry <- function(id, submit = NULL, root = NULL) {
+task_retry <- function(id, submit = NULL, resources = NULL, root = NULL) {
   root <- hipercow_root(root)
+  
+  # More thinking to do on what resources should be for a retry
```

Comment:
Can you make a ticket? we should probably chat through this one to find a reasonable solution

Reply:
Let's use yours - 
https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-4941/Resources-workflow-for-retried-tasks